### DISCUSSION TITLE
is it possible to prerender a spa application built in vite and tanstack/react-router  ?

### DISCUSSION THREAD

**Discussion by ymolists** [vikejs/vike#1881] [Discussion]
Hi.

I am currently using `@tanstack/react-router` and need to prerender all my pages to static html. I am not using ssr or anything like that. is it possible to use vike for just the prerendering part ? Meaning i already have a router.

I am hoping i dont have to do a big migration to integrate with this tool

Best Regards

**Comment by brillout** [comment]
See notes at https://vike.dev/react-router. It also applies for TanStack Router.
